src/tips/en/simulate-multiple-devices.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-01-06
+authors: Kilian Valkhof
+title: Work on multiple synced devices
+tags: ["productivity", "testing", "browser:polypane"]
+---
+
+Instead of [simulating devices one by one](/tips/en/simulate-devices/) and switching between them, Polypane supports testing on multiple simulated devices and viewports at the same time.
+
+It lets you test different simulated devices, viewports and media query features (like light and dark mode) side-by-side while keeping all your interactions (like clicks, hovers and keyboard input) in sync across all devices. This means anything you do in one device happens in all of them.
+
+
+![Polypane showing three different devices side-by-side.](/assets/img/simulate-devices-polypane.png)
```

Comment:
It looks like you forgot to check in this png file.

Reply:
because it's already in the repository, see it at this commit here: https://github.com/captainbrosset/devtools-tips/blob/ad30c750e09d39db52b34e3ba9a19b94f09aa0d0/src/assets/img/simulate-devices-polypane.png